src/io/thermo/reader.rs

```diff
@@ -864,11 +864,16 @@ pub(crate) mod sealed {
             event.injection_time = vevent.injection_time();
             let window = ScanWindow::new(vevent.low_mz() as f32, vevent.high_mz() as f32);
             event.scan_windows.push(window);
-            if let Some(cv) = vevent.compensation_voltage() {
-                let mut param =
-                    ControlledVocabulary::MS.param_val(1001581, "FAIMS compensation voltage", cv);
-                param.unit = Unit::Volt;
-                event.add_param(param);
+            if let Some(cv) = vevent.compensation_voltages() {
+                for cv in cv.iter() {
+                    let mut param = ControlledVocabulary::MS.param_val(
+                        1001580,
+                        "FAIMS compensation voltage",
+                        cv,
+                    );
+                    param.unit = Unit::Volt;
+                    event.add_param(param);
```

Comment:
This is sufficiently rare that I don't feel the need to make deep cut into the API surface yet, but repeating a `Param` multiple times is not ergonomic and all the facades around ion mobility will produce a valid-but-incorrect result.

Semantically, `ion mobility attribute` is-a `scan attribute` , and they are allowed to be repeated, but we have no formalism around how to handle the same term repeated. In all other places a ramp is found, the ramp is defined in terms of its start and end point by separate terms, e.g. `collision energy ramp start/end`, `scan window lower/upper limit`, etc.

Reply:
I've proposed a PSI term for FAIMS start and end, and added handling of that case, as well as pushing out a warning if there are more than 2 voltage offsets.